cl/alerts/tasks.py

```diff
@@ -100,6 +106,22 @@ def send_docket_alert(d_pk: int, since: datetime) -> None:
     connection = get_connection()
     connection.send_messages(messages)
 
+    # Notify all the webhooks available for the users subscribed
+    for user in users:
+        for webhook in user.webooks:
+            if webhook.enabled:
+                post_content = make_docket_alert_webhook_payload(d)
```

Comment:
At this point in the process, what is the best way to get ahold of the RecapDocument from earlier? Should I just pass it in? Or is there a clever way to get this from the docket entry?

Reply:
Also, I have set this up to presume we'll hit the webhook for every new entry, documents or not.